e2e/test/FaultInjectionPoolAmqpTests.MessageSendFaultInjectionPoolAmqpTests.cs

```diff
@@ -370,6 +371,68 @@ public async Task Message_IoTHubSak_ThrottledConnectionRecovery_SingleConnection
                 authScope: ConnectionStringAuthScope.IoTHub).ConfigureAwait(false);
         }
 
+        // TODO: #943 - Honor different pool sizes for different connection pool settings.
+        [Ignore]
```

Comment:
Same as above: we can split into different processes and enable everything until we get the named connection pools feature.

Reply:
@abhipsaMisra - Can you plz add this to backlog and check in fix?